backend/src/main/java/heartbeat/service/pipeline/buildkite/BuildKiteService.java

```diff
@@ -318,7 +318,7 @@ public DeployInfo mapToDeployInfo(BuildKiteBuildInfo buildInfo, List<String> ste
 		}
 
 		return new DeployInfo(buildInfo.getPipelineCreateTime(), job.getStartedAt(), job.getFinishedAt(),
-				buildInfo.getCommit(), job.getState(), job.getName());
+				buildInfo.getCommit(), job.getState(), "canceled".equals(buildInfo.getState()), job.getName());
```

Comment:
extract `cancelled` to constant, BTW correct spelling is `cancelled`

Reply:
actually canceled is America spell, and cancelled is English spell， they are both correct， and. the state from buildKiteFeign Client is canceled, so we'd better use canceled.
and already extract constant

Reply:
Make sense.